tcf_website/static/common/bind-sort.js

```diff
@@ -0,0 +1,12 @@
+import { sortByNumber, sortByRating, sortByDifficulty, sortByGpa, sortByRecency, sortByRecencyOnce } from "./sortClasses.js";
+
+if ($("#number-sort-btn").length) {
```

Comment:
Can you add a comment here explaining that these `if`s need to be here because `toolbar.html` is reused across pages and we need different sorting buttons on dept/course?

Reply:
Sounds good, I'll be sure to add that in.